examples/community-detection.ipynb

```diff
@@ -0,0 +1,1148 @@
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>SUBREDDIT</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>leagueoflegends</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>theredlion</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>inlandempire</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>nfl</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>playmygame</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         SUBREDDIT\n",
+       "0  leagueoflegends\n",
+       "1       theredlion\n",
+       "2     inlandempire\n",
+       "3              nfl\n",
+       "4       playmygame"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# get unique nodes for each column\n",
+    "source_nodes = pd.Series(df['SOURCE_SUBREDDIT']).unique()\n",
+    "target_nodes = pd.Series(df['TARGET_SUBREDDIT']).unique()\n",
+    "\n",
+    "# get unique nodes for both columns\n",
+    "all_nodes = pd.Series(pd.concat([df['SOURCE_SUBREDDIT'], df['TARGET_SUBREDDIT']])).unique()\n",
+    "\n",
+    "# create new dataframe with distinct nodes\n",
+    "nodes_df = pd.DataFrame({'SUBREDDIT': all_nodes})\n",
+    "nodes_df.head()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c30a4378",
+   "metadata": {},
+   "source": [
+    "Finally, we load this data (nodes and edges) into a Graph Database and a GDS graph."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "0b18e522",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "driver = GraphDatabase.driver(NEO4J_CONNECTION_URI,\n",
+    "    auth=(NEO4J_USERNAME, NEO4J_PASSWORD))\n",
+    "\n",
+    "# Create nodes and relationships in the graph using UNWIND\n",
+    "with driver.session() as session:\n",
+    "    # Create nodes using UNWIND\n",
+    "    nodes_list = nodes_df.to_dict('records')\n",
+    "    session.run(\"UNWIND $nodes_list AS node_props CREATE (n:Subreddit {node_id: node_props.SUBREDDIT, node_label: node_props.SUBREDDIT})\", nodes_list=nodes_list)\n",
```

Comment:
How about using `name` instead of `node_id` for the property here as its easier to now what it represents.

Also you are setting both `node_id` and `node_label` here. Why both?

Reply:
Oh yeah. Made those changes. That was just lazy from my end.